common/openapi.yml

```diff
@@ -90,6 +90,29 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/oidc/v1/{provider}/access-token:
+    parameters:
+      - $ref: '#/components/parameters/userIdParam'
```

Comment:
We won't need the userId as a param, as we'll be able to resolve that from the request headers. Looks the controllers use `var samUser = samUserFactory.from(request);` to get the Sam user.